arduinobot_sim:

  joint_1_position_controller:
    type: "position_controllers/JointPositionController"
    joint: joint_1
  
  joint_2_position_controller:
    type: "position_controllers/JointPositionController"
    joint: joint_2
  
  joint_3_position_controller:
    type: "position_controllers/JointPositionController"
    joint: joint_3
  
  joint_4_position_controller:
    type: "position_controllers/JointPositionController"
    joint: joint_4

  joint_5_position_controller:
    type: "position_controllers/JointPositionController"
    joint: joint_5
